python/cudf/cudf/util/internalutil.py

```diff
@@ -265,6 +265,67 @@ def compare_and_get_name(a, b):
     return None
 
 
+def hash_pandas_object(
```

Comment:
Is the reason cudf needs a `hash_pandas_object` because of dask calls to `hash_pandas_object`?

Reply:
We only need this if we choose to use the `sys.modules` approach of compatibility (which I'm somewhat against).  I think that we should wait to see what @TomAugspurger prefers on this comment:

https://github.com/dask/dask/pull/5184#issuecomment-516625743